.travis.yml

```diff
@@ -55,7 +55,7 @@ jobs:
         - conda install $CHANS nbsite==0.5.1 sphinx_ioam_theme
         - doit env_capture
       script:
-        - doit before_test  ## NOTE do not run this when building the site locally!!
+        - git checkout evaluated doc  # all large evaluted notebooks and their json files should be checked in to this branch
```

Comment:
```suggestion
        - git checkout evaluated doc  # all large evaluated notebooks and their json files should be checked in to this branch
```

Reply:
What's the difference between an evaluated notebook and the json file? Isn't that the same thing?

Reply:
whoops. I missed this comment. Holomaps have related json blobs and these are generated when the notebook is evaluated.